src/main/java/nus/climods/logic/parser/CliModsParser.java

```diff
@@ -39,6 +40,8 @@ public static Command parseCommand(String userInput) throws ParseException {
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
+        case ExitCommand.COMMAND_WORD:
+            return new ExitCommandParser().parse(commandWord);
```

Comment:
Should we check for the `arguments` instead since we alr checked the `commandWord`?

Reply:
I think dunnid coz was thinking if the user accidentally did a keyboard spam, the exit command still allow users to exit.  e.g. exit 123 will still exit the program. However, exit123 will not exit the program since it is a wrong command.